brainstormforce/ultimate-addons-for-gutenberg
# fix: allow links to be clickable within FAQ sections

### Description

Allow links to be clickable within FAQ sections. The e.preventDefault(); in the FAQ click handler was preventing links from being opened as normal.

### Types of changes




Bug fix (non-breaking change which fixes an issue).

### How has this been tested?

- Click the link. Expected behavior: link opens, faq section stays open. 
- Click on the open FAQ section but not the link. Expected behavior: faq section closes. Link is not open.

### Checklist:
